geomstats/visualization/stiefel_manifold.py

```diff
@@ -0,0 +1,450 @@
+"""Stiefel manifold module."""
+import matplotlib.pyplot as plt
+import numpy as np
+
+import geomstats.backend as gs
+from geomstats.geometry.hypersphere import Hypersphere
+from geomstats.visualization import Sphere
+
+# %%
+"""
+    Consider <n=3, p=1> Stiefel Manifold.
+    Dimension is 2, which can be represented as sphere.
+"""
+
+
+class StiefelSphere:
+    """
+    Class used to plot something in Stiefel shape space of 2D triangles.
+
+    offering a 3D visualization of Kendall shape space of order
+    (3,1), and its related objects.
+
+    Attributes
+    ----------
+      ax : NoneType
+      Axis of the stiefel object
+
+    References
+    ----------
+    Absil, Pierre-Antoine. "Stiefel Manifolds and their Applications." (2009)
```

Comment:
Format this reference, using examples from other docstring in geomstats' code.
Otherwise, this will not render properly on the documentation website of geomstats, geomstats.ai